### DISCUSSION TITLE
Please don't take back your star without telling me why!

### DISCUSSION THREAD

**Discussion by juzzlin** [juzzlin/Heimer#258] [Announcements]
I occasionally see stars disappearing and I'd like to KNOW IF THERE'S A PROBLEM WITH HEIMER! Instead of just removing your star PLEASE REPORT A BUG! I cannot test anything and might not even know if something is broken or doesn't work!

Thank you!

**Comment by peter88213** [comment]
I can only speak for myself: sometimes I set stars as bookmarks while browsing around to later rediscover a repository that looks interesting. Every now and then I tidy up the list. This should not worry you and has nothing to do with the quality of your work. 

Cheers, 
Peter
